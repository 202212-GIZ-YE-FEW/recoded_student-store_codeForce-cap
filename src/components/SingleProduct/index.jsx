import SingleProduct from "../../pages/single-product"
export default SingleProduct
