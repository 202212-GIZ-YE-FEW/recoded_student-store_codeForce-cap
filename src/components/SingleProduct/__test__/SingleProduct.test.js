import renderer from "react-test-renderer"

import SingleProduct from ".."

it("renders correctly", () => {
  const tree = renderer.create(<SingleProduct />).toJSON()
  expect(tree).toMatchSnapshot()
})
