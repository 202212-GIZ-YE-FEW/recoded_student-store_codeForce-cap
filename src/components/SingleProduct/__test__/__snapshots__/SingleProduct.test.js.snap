// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="max-w-[1250px] mx-auto my-[50px]"
>
  <section
    className="py-12 sm:py-16"
  >
    <div
      className="container mx-auto px-4"
    >
      <div
        className="lg:col-gap-12 xl:col-gap-16 grid grid-cols-1 gap-12 lg:mt-12 lg:grid-cols-4 lg:gap-16 justify-items-center"
      >
        <div
          className="lg:col-span-2 lg:row-end-1"
        >
          <div
            className="md:row"
          >
            <div
              className="md:col-span-1 lg:order-2 lg:ml-5"
            >
              <div
                className="overflow-hidden rounded-lg"
              >
                <img
                  alt=""
                  className="h-full w-full max-w-full object-cover"
                  data-nimg="1"
                  decoding="async"
                  height={500}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="photo-one.svg"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={500}
                />
              </div>
            </div>
            <div
              className="md:col-span-1 mt-2 w-full lg:order-1 lg:w-full justify-between"
            >
              <div
                className="flex lg:flex-row lg:items-start lg:w-full md:flex-none sm:flex-row sm:items-start sm:w-full"
              >
                <button
                  className="flex-0 aspect-square mb-3 h-32 w-full overflow-hidden rounded-lg border-2 border-transparent text-center"
                  type="button"
                >
                  <img
                    alt=""
                    className="h-full w-full object-cover"
                    data-nimg="1"
                    decoding="async"
                    height={100}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="photo-two.svg"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={100}
                  />
                </button>
                <button
                  className="flex-0 aspect-square mb-3 h-32 w-full overflow-hidden rounded-lg border-2 border-transparent text-center"
                  type="button"
                >
                  <img
                    alt=""
                    className="h-full w-full object-cover"
                    data-nimg="1"
                    decoding="async"
                    height={100}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="photo-three.svg"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={100}
                  />
                </button>
                <button
                  className="flex-0 aspect-square mb-3 h-32 w-full overflow-hidden rounded-lg border-2 border-transparent text-center"
                  type="button"
                >
                  <img
                    alt=""
                    className="h-full w-full object-cover"
                    data-nimg="1"
                    decoding="async"
                    height={100}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="photo-four.svg"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={100}
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          className="lg:col-span-2 lg:row-span-2 lg:row-end-2"
        >
          <h1
            className="text-2xl font-bold text-[#7874F2] bg-background-title bg-no-repeat sm:text-3xl"
          >
            3-Seat Sofa, Gray
          </h1>
          <div
            className="mt-5 items-center font-poppins pl-4"
          >
            <p
              className="text-1xl text-[#585785]"
            >
              <span
                className="font-bold"
              >
                Category: 
              </span>
              Furniture
            </p>
            <p
              className="text-1xl text-[#585785]"
            >
              <span
                className="font-bold"
              >
                Condition: 
              </span>
              Used
            </p>
          </div>
          <h2
            className="mt-8 font-poppins text-[#585785] text-2xl pl-4"
          >
            Details:
          </h2>
          <hr
            className="border-2 ml-4 bg-[#7874F2]"
          />
          <div
            className="flex flex-col text-[#585785] items-center justify-between space-y-4 py-4 sm:flex-row sm:space-y-0"
          />
          <div
            className="font-poppins text-[#585785] text-1xl pl-4"
          >
            <p>
              Iâ€™ve been scratching this sofa for 2 years. Selling due to boredom. Bought it from Ikea, model name is : EKTORP. No bedbugs, cleaned it.
            </p>
            <p>
              <span
                className="block"
              >
                Dimensions:
              </span>
               Width: 218 cm ,Depth: 88 cm, Height: 88 cm, Seat depth: 49 cm, Seat height: 45 cm.
            </p>
          </div>
          <div
            className="mt-10 flex select-none flex-wrap items-center gap-1"
          >
            <div
              className="flex flex-row bg-[#585785] w-64 h-20 lg:w-96 lg:h-36 md:w-96 md:h-36 rounded-l-full -mx-1 border-r-4 border-dashed"
            >
              <div
                className="m-auto ml-2"
              >
                <img
                  alt=""
                  className="w-16 h-16 lg:w-32 lg:h-28 md:w-32 md:h-28"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="cat-photo.svg"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={130}
                />
              </div>
              <div
                className="m-auto lg:m-auto font-poppins text-white text-[14px] lg:text-xl md:text-xl"
              >
                <h1
                  className="font-semibold text-1xl"
                >
                  Rifik Haspolat
                </h1>
                <h2>
                  <span
                    className="font-bold"
                  >
                    email:
                  </span>
                   miyav@gmail.com
                </h2>
                <h2>
                  <span
                    className="font-bold"
                  >
                    Location:
                  </span>
                   istanbul/turkey
                </h2>
              </div>
            </div>
            <div
              className="grid justify-items-center bg-orange-500 w-20 h-20 lg:w-36 lg:h-36 md:w-36 md:h-36"
            >
              <h1
                className="font-poppins text-white font-semibold text-[14px] lg:text-3xl md:text-3xl m-auto"
              >
                150
                <span>
                  $
                </span>
              </h1>
            </div>
          </div>
        </div>
      </div>
      <div
        className="mt-20 w-full object-cover grid justify-items-center"
      >
        <img
          alt=""
          data-nimg="1"
          decoding="async"
          height={350}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="map.svg"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={1400}
        />
      </div>
    </div>
  </section>
</section>
`;
